website/src/components/Messages/BaseMessageEntry.tsx

```diff
@@ -2,19 +2,21 @@ import { Avatar, AvatarProps, Box, BoxProps, Flex, useColorModeValue } from "@ch
 import { forwardRef, lazy, Suspense } from "react";
 import { colors } from "src/styles/Theme/colors";
 import { StrictOmit } from "ts-essentials";
-
+import { PluginUsageDetails } from "./PluginUsageDetails";
 const RenderedMarkdown = lazy(() => import("./RenderedMarkdown"));
 
 export type BaseMessageEntryProps = StrictOmit<BoxProps, "bg" | "backgroundColor"> & {
   content: string;
   avatarProps: Pick<AvatarProps, "name" | "src">;
   bg?: string;
   highlight?: boolean;
+  usedPlugin?: object;
+  isAssistant?: boolean;
```

Comment:
I am not really sure if we want this to be so intrusive into the component structure.

Reply:
How to get usedPlugin info inside of a message, if you have idea, i can modify it?

Reply:
we show below the message a lot of info, thumbs up & down, and the message parameters, is it possible to show this stuff there?

Reply:
Not really. This is more like a tool, for plugin consumer/developer.
Its not like just some verbose data.

Reply:
<img width="953" alt="Screenshot 2023-04-30 at 19 37 31" src="https://user-images.githubusercontent.com/13547364/235367797-d4717a05-2062-49ae-90d1-e5dde6e90b7b.png">

I mean on this component